src/evaluate.jl

```diff
@@ -261,6 +268,31 @@ function evaluate{T<:NumberNotSeries}(a::TaylorN{Taylor1{T}},
     return suma
 end
 
+function evaluate{T<:Number,S<:NumberNotSeries}(a::TaylorN{T},
+        vals::Array{TaylorN{S},1})
+    @assert length(vals) == get_numvars()
+
+    num_vars = get_numvars()
+    ct = coeff_table
```

Comment:
This creates a copy of `coeff_table`. Perhaps using a `view` of it, or directly `coeff_table` saves this.

I didn't notice this before, sorry, but I think this should also be changed in other occurrences.